pkg/cmd/kubernikusctl/common/openstack.go

```diff
@@ -44,6 +44,9 @@ func (o *OpenstackClient) BindFlags(flags *pflag.FlagSet) {
 	flags.StringVar(&o.Scope.ProjectName, "project-name", o.Scope.ProjectName, "Scope to this project. Also requires --project-domain-name/--project-domain-id [OS_PROJECT_NAME]")
 	flags.StringVar(&o.Scope.DomainID, "project-domain-id", o.Scope.DomainID, "Domain of the project [OS_PROJECT_DOMAIN_ID]")
 	flags.StringVar(&o.Scope.DomainName, "project-domain-name", o.Scope.DomainName, "Domain of the project [OS_PROJECT_DOMAIN_NAME]")
+	flags.StringVar(&o.ApplicationCredentialName, "application-credential-name", o.ApplicationCredentialName, "Project application credential name [OS_APPLICATION_CREDENTIAL_NAME]")
+	flags.StringVar(&o.ApplicationCredentialID, "application-credential-id", o.ApplicationCredentialName, "Project application credential id [OS_APPLICATION_CREDENTIAL_ID]")
+	flags.StringVar(&o.ApplicationCredentialSecret, "application-credential-secret", o.ApplicationCredentialSecret, "Project application credential secret [OS_APPLICATION_CREDENTIAL_SECRET]")
```

Comment:
The default for this variable should be `""`. Because otherwise `--help` will print out the secret when its set via environment variable.

Reply:
@databus23 I changed this and also simplified validation. Please have a look.